pkg/liquidity-mining/contracts/gauges/ChildChainGauge.vy

```diff
@@ -1,4 +1,4 @@
-# @version 0.3.1
+# @version 0.3.3
```

Comment:
Incrementing the version by 2?

Reply:
This is the Vyper compiler version.
I need 0.3.3 in particular to be able to use 'dynamic' arrays. This compiler version is [already listed](https://github.com/balancer/balancer-v2-monorepo/blob/45bfdc266fbff3b3b79cc68af7f4193b6ebd2124/pkg/liquidity-mining/hardhat.config.ts#L21) in the compilers we use, so we don't need to modify the general configuration for this to work.